app/scripts/services/tile-proxy.js

```diff
@@ -158,11 +124,37 @@ export const setTileProxyAuthHeader = (newHeader) => {
 export const getTileProxyAuthHeader = () => authHeader;
 
 /**
```

Comment:
Some context in the docstring here would be helpful. What does this function do? Under what circumstances? Why?

Reply:
Added an comment and unit test, thanks for the suggestion!